src/createServer.js

```diff
@@ -1,3 +1,51 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleFaviconRequest } = require('./faviconRequest');
+const { validateRequest } = require('./validatedRequest');
+const { createErrorResponse } = require('./errorResponse');
+const { createSuccessResponse } = require('./successResponse');
+const { createJsonResponse } = require('./jsonPesponse');
+
+const PORT = 4000;
```

Comment:
As i remember, we don't need to hardcode our PORT, we are hardcoding in in `main.js`

Reply:
Oh, I didnt see this file) just write it to work on my laptop, not exacly understant how it should be in the best way)